Loop/Models/RemoteCommand.swift

```diff
@@ -70,10 +70,27 @@ extension RemoteCommand {
                 absorptionTime = TimeInterval(hours: absorptionOverride)
             }
             let quantity = HKQuantity(unit: .gram(), doubleValue: carbsValue)
-            let newEntry = NewCarbEntry(quantity: quantity, startDate: Date(), foodType: "", absorptionTime: absorptionTime)
+            
+            var startDate = nowDate
+            if let createdAtStr = notification["created-at"] as? String {
```

Comment:
'created-at' is generally a term in Loop that we use to record the date that a record was created at, which can be different from the time of eating. The 'startTime' is the the time the user entered, to represent the time of eating.

Reply:
Thanks - This has been updated.